ILTransform/ILRewriter.cs

```diff
@@ -426,7 +426,10 @@ private void RewriteFile(string source)
                     rewriteLine = _testProject.LastHeaderCommentLine;
                 }
                 rewriteLine++;
-                lines.Insert(rewriteLine++, "using Xunit;");
+                if (!lines.Any(l => l.Contains("using Xunit")))
```

Comment:
Just replace the definition of `usingXUnit` above with this.

This has the potential for false positives.. you can refine it by adding the semicolon, using regex (`^\s*using\s+Xunit\s*(?://,*)?;`), or more, but of course there's no need until such a case is hit